src/pages/u/[handle].tsx

```diff
@@ -4,14 +4,51 @@ import withAccessType from '@/lib/hoc/withAccessType';
 import { CookieService, PermissionService } from '@/lib/services';
 import type { PublicProfile } from '@/lib/types/apiResponses';
 import { CookieType } from '@/lib/types/enums';
+import defaultProfilePicture from '@/public/assets/profile-pics/adorable0.png';
+import styles from '@/styles/pages/u.module.scss';
+import Image from 'next/image';
 import type { GetServerSideProps } from 'next/types';
 
 interface UserProfilePageProps {
   user: PublicProfile;
 }
 
 const UserProfilePage = ({ user }: UserProfilePageProps) => {
-  return <pre>{JSON.stringify(user, null, 2)}</pre>;
+  // return <pre>{JSON.stringify(user, null, 2)}</pre>;
+  return (
+    <div className={styles.profilePage}>
+      <div className={styles.cardWrapper}>
+        <div className={styles.banner} />
+        <div className={styles.profileCard}>
+          <div className={styles.cardHalf}>
+            <div className={styles.horizontal}>
+              <Image
+                src={user.profilePicture ?? defaultProfilePicture}
```

Comment:
Something to think about if you want to include it in this PR with a utility function: How can you randomly provide an image for all ACM users that don't have one uploaded out of the full set of 9 (i think) default images we have